packages/block-editor/src/store/reducer.js

```diff
@@ -1913,6 +1913,13 @@ export function blockEditingModes( state = new Map(), action ) {
 	return state;
 }
 
+export function openedBlockSettingsMenu( state = null, action ) {
```

Comment:
The block editor reducer is quite thoroughly covered by unit tests. A couple of simple tests could be useful to cover this new reducer subtree.